README.md

```diff
@@ -1,19 +1,43 @@
 # Maestro Cloud Action
 
-Run your Flows on [Maestro Cloud](https://cloud.mobile.dev).
+Run your Flows on [Robin](https://www.robintest.com/) or [Maestro Cloud](https://cloud.mobile.dev).
 
-# Using the action
+## Using the action
 
 Add the following to your workflow. Note that you can use the `v1` tag if you want to keep using the latest version of the action, which will automatically resolve to all `v1.minor.patch` versions as they get published.
 
 ```yaml
 - uses: mobile-dev-inc/action-maestro-cloud@v1.9.4
   with:
-    api-key: ${{ secrets.MAESTRO_CLOUD_API_KEY }}
+    api-key: ${{ secrets.ROBIN_API_KEY }}
+    project-id: 'proj_01abc3def456ghij78klmnopq9'
```

Comment:
```suggestion
    project-id: 'proj_01abc3def456ghij78klmnopq9' # replace this with your actual project id
```

Reply:
What about this in the first instance, then just the example ID in the remainder?
You're right in that my existing example takes a little parsing to notice that it's not a real ID.

```suggestion
    project-id: 'proj_01example0example1example2' # replace this with your actual project id
```

Reply:
That sounds great!